libkernelflinger/security.c

```diff
@@ -207,6 +283,24 @@ EFI_STATUS init_rot_data(UINT32 boot_state)
     /* TBD: keyHash should be the key which used to sign vbmeta.ias */
     memset_s(rot_data.keyHash256, SHA256_DIGEST_LENGTH, 0, SHA256_DIGEST_LENGTH);
 
+    rot_data.brandSize = 0;
+    memset_s(rot_data.brand, ATTESTATION_ID_MAX_LENGTH, 0, ATTESTATION_ID_MAX_LENGTH);
```

Comment:
why add the id's informantions into the rot_data structure? they're not belongs to the rot, can you move them into another defined structure?  also suggest not reuse the function update_rot_data(), try to add another function such as update_attestation_ids() or get_attestation_ids()